packages/cardano-services/src/Asset/CardanoTokenRegistry.ts

```diff
@@ -0,0 +1,104 @@
+import { Asset, Cardano } from '@cardano-sdk/core';
+import { Logger } from 'ts-log';
+import { TokenMetadataService } from './types';
```

Comment:
The interface would be better sourced in the `core` package, although I'm not sure where. Maybe `core/src/Asset/types/TokenMetadataService.ts`

Reply:
@rhyslbw do you have some other use case of this interface in mind? This is not the first case when `util-services`-like package would be useful (`RunnableModule` is another example).

If we move this then we should also move `NftMetadataService` as they are quite similar.

If there's no immediate need, I'd say just leave them in `cardano-services` until some other package needs to use them.

Reply:
More or less I had the same thoughts as @mkazlauskas

Reply:
No other use-cases at the moment, so let's leave it here